containers/unit_tests/TestDynamicView.hpp

```diff
@@ -83,7 +83,8 @@ struct TestDynamicView {
           },
           result_sum);
 
-      ASSERT_EQ(result_sum, (value_type)(da_size * (da_size - 1) / 2));
+      ASSERT_EQ(result_sum,
+                static_cast<value_type>(da_size) * (da_size - 1) / 2);
```

Comment:
Again changes possibly value (overflowing mantissa), but if we do so the original summation would have overflown in a different way already and the test wouldn't work - so it's fine.